# Generated by Django 2.1.1 on 2020-04-29 19:32

import datetime
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='customuser',
            name='addressUser',
            field=models.CharField(blank=True, max_length=50),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='dateLastAccessUser',
            field=models.DateField(blank=True, default=datetime.date.today),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='dateOfBirthUser',
            field=models.DateField(blank=True, default=datetime.date.today),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='genderUser',
            field=models.CharField(blank=True, max_length=20),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='phoneUser',
            field=models.CharField(blank=True, max_length=10, unique=True, validators=[django.core.validators.RegexValidator(message='Numero incorrecto', regex='^\\+?1?\\d{7,10}$')]),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='rhUser',
            field=models.CharField(blank=True, max_length=20),
        ),
        migrations.AlterField(
            model_name='teacheruser',
            name='degreesTeacher',
            field=models.CharField(blank=True, max_length=100),
        ),
    ]
